examples/locks/locks.bal

```diff
@@ -10,29 +10,29 @@ public function main() {
 }
 
 type Counter object {
-    //Shared field among same object instances.
+    // The `count` field will be shared among the same object instances.
```

Comment:
Doesn't this send the wrong impression that `count` field is common to all instances of `Counter` object type?

Reply:
I guess the intended meaning here is that an object of `Counter` type will be shared among multiple workers and those concurrent workers would access and attempt to modify the `count` field.

Reply:
Addressed with 259c088